### PR TITLE
Error handled while user trying to remove the contact

### PR DESCRIPTION
**Pull request by enigmaticmahesh**
## Summary

Error was not handled when user tries to remove the contact in its profile, which has been fixed in this PR. This issue was found while debugging the current issue

### CODE REVIEW

**Review by mdshamoon — COMMENTED** [review]
@enigmaticmahesh adding the error handler on the delete action is fine. But the actual issue was that the user should not have the access to delete or remove the NGO main account number.

basically we need to hide the remove action on the NGO main account number. Right now its hidden only if the number is of current user,
image

**Review by kurund — COMMENTED** [review]
@mdshamoon 
I think we checked, and other users do not have access to delete NGO's primary contact.

@enigmaticmahesh 
Maybe you should make another user account with the role of admin and try deleting the main account?

**Comment by mdshamoon** [comment]
> and other users do not have access to delete NGO's primary contact.

We don't allow deleting your own contact. So yes I think we need to login with a different admin account.

**Comment by enigmaticmahesh** [comment]
Restricted the user to delete the contact by hiding the Remove button in this [Commit](https://github.com/glific/glific-frontend/pull/2106/commits/a2e9c2fc420d518dd32eac9b46425b125b1d2ad5) @mdshamoon

File: src/containers/Profile/Profile.tsx [hunk 1]
```diff
@@ -58,6 +59,7 @@ export const Profile: React.SFC<ProfileProps> = ({
 
   const loggedInUserContactId = data.currentUser.user.contact.id;
   const currentUserPhone = data.currentUser.user.phone;
+  const organizationPhone = data?.currentUser?.user?.organization?.contact?.phone;
```
Comment (mdshamoon) [inline src/containers/Profile/Profile.tsx:62]:
```suggestion
  const { user } = data.currentUser;
  const loggedInUserContactId = user.contact.id;
  const currentUserPhone = user.phone;
  const organizationPhone = user?.organization?.contact?.phone;
```

Comment (enigmaticmahesh) [reply src/containers/Profile/Profile.tsx]: @mdshamoon implemented your suggestion

**Review by mdshamoon — APPROVED** [review]
@enigmaticmahesh changes looks good. Please fix the deepscan issue and merge.

**Pull request merged**